samples/application-conf/application.properties

```diff
@@ -125,5 +125,13 @@ testcasePgmRule =   {"category": "PROGRAMDEPENDENCY", \
               ] \
             }
 
+# Rule to locate Easytrieve mactos
```

Comment:
typo. should be macros instead of mactos

Reply:
Thanks @BeckerFrank I have migrated my fork to support the new way of the copySourceFiles and have fixed this typo as well.